sapphire/utils.py

```diff
@@ -20,7 +20,7 @@
 ERR = [-1, -999]
 
 #: Speed of light in vacuum in m / ns.
-c = 0.299792458
```

Comment:
The `a`, `b`, `c`, etc variables are used in a method that also uses `c`. Perhaps the `c` should be skipped, or something. Ok, I will have a look at it tonight, and revert `C` back to `c`